.phrozn/entries/index.twig

```diff
@@ -45,5 +45,10 @@ events:
         - name: NFQ
           url: http://www.nfq.lt
           logo: nfq-logo.png
+      speakers:
+        - name: Justas Butkus
+          title: Palaikykite mano alų
```

Comment:
Nesu tikras ar temos pavadinimas tinkamas pagal įsigaliojusius LR įstatymų pakeitimus.

Reply:
References to illegal substances were removed, adhering to strict policies. Will be happy to go and have some herbal tea after the talks are over.